frontend/public/components/graphs/area.tsx

```diff
@@ -53,9 +58,20 @@ export const AreaChart: React.FC<AreaChartProps> = ({
   xAxis = true,
   yAxis = true,
   chartStatus,
+  byteDataType = '',
 }) => {
   const [containerRef, width] = useRefWidth();
-  const getLabel = ({ datum: { x, y } }) => `${humanize(y).string} at ${formatDate(x)}`;
+  let processedData = data;
+  let unit: string = '';
+
+  //If user has speicified a type for incoming data is byte related
+  if (byteDataType) {
+    const result = processFrame(data, byteDataType);
+    processedData = result.processedData;
+    unit = result.unit;
+  }
+  const getLabel = getLabelFunction(humanize, formatDate, byteDataType, unit);
+  const tickFormat = getTickFunction(humanize, byteDataType, unit);
```

Comment:
Again, don't bypass the humanize callback. Use as intended. If we want to keep the units in the same order of magnitude for tick values, we can also provide the 3rd `desiredUnit` arg to the humanize prop. I seem to remember some discussion about how the y-axis tick values should behave when they span an order of magnitude, but I don't remember where it landed. Regardless, the humanize function can handle either scenario:

**Automatically convert units to 'best value' (units are automatically scaled):**
```js
  const tickFormat = React.useCallback((tick) => humanize(tick, unit).string, [humanize, unit]);
```

**'Lock' units in to be all the same:**
```js
  const tickFormat = React.useCallback((tick) => humanize(tick, unit, unit).string, [humanize, unit]);
```